aten/src/ATen/native/cpu/ScatterGatherKernel.cpp

```diff
@@ -627,7 +628,7 @@ void cpu_scatter_reduce_expanded_index(const Tensor& self, const Tensor& index,
 
   int64_t* sorted_col_index_keys = nullptr;
   int64_t* sorted_col_index_values = nullptr;
-  std::tie(sorted_col_index_keys, sorted_col_index_values) = radix_sort_parallel(
+  std::tie(sorted_col_index_keys, sorted_col_index_values) = fbgemm::radix_sort_parallel(
```

Comment:
So does this mean that if FBGEMM were not available, this code would no longer work?

Reply:
Yes. `_expanded_index` requires parallelized `radix_sort` to work efficiently, if that's not met, a regular execution path is chosen. Reuse of fbgemm code (instead of local copy) was also requested in original implementation - please see [this comment](https://github.com/pytorch/pytorch/pull/82703/files#r1044538418).